data/remnant/remnant outfits.txt

```diff
@@ -276,6 +277,133 @@ outfit "EMP Torpedo Bay"
 		"missile strength" 80
 	description "Electromagnetic pulse weapons were developed by the Remnant during the early days of their colony, when they were living in fear that the Alphas would overrun human space and expand beyond it."
 
+
+
+outfit "Teciimach Canister"
+	category "Ammunition"
+	licenses
+		Remnant
+	cost 35500
+	thumbnail "outfit/emp torpedo"
+	"mass" 1
+	"teciimach canister capacity" -1
+	description "This canister contains everything required to assemble a Teciimach torpedo. These weapons emit strong electromagnetic pulses which interfere with a ship's generators and neutralize a portion of their energy."
+	description `	Unlike the previous generation of "EMP torpedoes," the Teciimach torpedoes actively shape their emission pattern to avoid impacting their allies.`
+
+outfit "Teciimach Canister Rack"
+	category "Ammunition"
+	licenses
+		Remnant
+	cost 298000
+	thumbnail "outfit/emp rack"
+	"mass" 5
+	"outfit space" -9
+	"energy capacity" 500
+	"teciimach canister capacity" 9
+	ammo "Teciimach Canister"
+	description "The Teciimach Canister rack is used to store extra sets of components, both to expand the capacity of a ship, and on carriers for resupplying fighters."
+	description "	In addition to its ammunition capacity, the storage bay can also contain a small amount of charge, enough to let an otherwise battery-less ship assemble Teciimach torpedoes."
+
+outfit "Teciimach Bay"
+	category "Secondary Weapons"
+	licenses
+		Remnant
+	cost 5250000
```

Comment:
I'm happy with the ammo cost, but I actually think the bay cost is way too high now. For comparison, the Finisher Pod, which has an ammo cost that is now similar to the Teciimach (32k vs 35.5k), only has a bay cost of 1.37 mil compared to the Teciimach's 5.25 mil.

A bay cost in the 900k to 1.2 mil range feels more appropriate to me, and the pod cost could be roughly half that.

Reply:
Sure, that sounds fine. I'll make the change.

Reply:
Change has been made. Thanks for the suggestions!